### DISCUSSION TITLE
애버커스 프레임워크 토론장에 오신것을 환영합니다~

### DISCUSSION THREAD

**Discussion by seongpil0948** [seongpil0948/framework-next#4] [Announcements]
## 👋 Welcome!
우리는 이곳을 커뮤니티의 소속에 상관없이 토론을 나눌 수 있는 장소로 사용하고 있습니다.
  * 궁금한 점을 질문해보세요.
  * 아이디어를 공유하세요.
  * 다른 커뮤니티 구성원과 교류하세요.
  * 다른 사람들을 환영하고 개방적입니다. 이 커뮤니티는 우리가 함께 만들어가는 커뮤니티라는 점을 기억하세요 💪.

 시작하려면 아래에 자신에 대한 소개와 함께 댓글을 달고 이 커뮤니티에서 어떤 일을 하는지 알려주세요.

**Comment by jun0108** [comment]
안녕하세요 퍼블리셔 **방준영**(26세)입니다.
공통 컴포넌트 퍼블리싱하러 왔어요😃

**Reply by seongpil0948** [reply]
반갑습니다 준영님 ^^ 잘 부탁드려요
